tools/include/eosio/gen.hpp

```diff
@@ -300,8 +304,8 @@ struct generation_utils {
                   return true;
                } else {
                   for ( auto name : names )
-                     if ( const auto* decl = rt->getDecl() ) {
-                        return decl->getName().str() == name;
+                     if ( rt->getDecl()->getName().str() == name ) {
```

Comment:
I don't understand why this changed.  `rt->getDecl()` can't guarantee that the decl pointer is non-null.

Reply:
Hmm, I don't remember touching this so maybe it was a bad merge. Regardless, I'll revert to the original form.